### PR TITLE
Feat/threshold example

### PR DESCRIPTION
**Pull request by tim775**

### CODE REVIEW

File: .github/workflows/examples_test.yml [hunk 1]
```diff
@@ -221,3 +221,59 @@ jobs:
...
+        run: >-
+          infracost breakdown --path=examples/thresholds/code/plan.json
+          --format=json --out-file=/tmp/infracost.json
+      - name: Calculate Cost Change
+        id: cost-change
+        uses: actions/github-script@v5
+        with:
+          script: >
+            // Read the breakdown JSON and get the past and current total
+            monthly costs
```
Comment (aliscott) [inline .github/workflows/examples_test.yml:243]: Does this being on a new line cause any issues?

File: .github/workflows/examples_test.yml [hunk 2]
```diff
@@ -221,3 +221,59 @@ jobs:
...
+            const current = breakdown.totalMonthlyCost;
+
+
+            // Calculate the percent and $ diffs
+
+            let absolutePercentChange = 0;
+
+            let absoluteCostChange = 0         ;   
+
+            if (past != 0) {
```
Comment (aliscott) [inline .github/workflows/examples_test.yml:258]:
```suggestion
            if (past !== 0) {
```

File: .github/workflows/examples_test.yml [hunk 3]
```diff
@@ -221,3 +221,59 @@ jobs:
...
+            }            
+
+            absoluteCostChange = Math.abs(past - current);
+
+
+            // Set the calculated diffs as outputs to be used in future steps
+
+            core.setOutput('absolute-percent-change', absolutePercentChange);
+
+            core.setOutput('absolute-cost-change', absoluteCostChange);
```
Comment (aliscott) [inline .github/workflows/examples_test.yml:269]: Are we wanting non-absolute numbers as well?

File: .github/workflows/examples_test.yml [hunk 4]
```diff
@@ -221,3 +221,59 @@ jobs:
...
+
+            let absolutePercentChange = 0;
+
+            let absoluteCostChange = 0         ;   
+
+            if (past != 0) {
+              absolutePercentChange = 100 * Math.abs((current - past) / past);
+            }            
+
+            absoluteCostChange = Math.abs(past - current);
```
Comment (aliscott) [inline .github/workflows/examples_test.yml:262]: Should we use the abs of the `diffMonthlyCost` here?

**Review by aliscott — APPROVED** [review]
This is great. @alikhajeh1 might want a bash example but I think this is a lot more readable.

**Comment by alikhajeh1** [comment]
This is awesome! No need for bash examples. It would be great if this thing logged what it's doing or it did in the GHA - here's what I see just now, expanding that just shows the code
Screen Shot 2021-11-24 at 6 05 34 PM

**Pull request merged**